app/views/layouts/application.html.erb

```diff
@@ -53,12 +53,16 @@
         </form> -->
         <ul class="nav navbar-nav navbar-right">
           <% if current_user %>
+            <% unless current_user.subscribed %>
+              <li><a href="/account">Subscribe to Pro</a></li>
+              <% else %>
+                <li><%= link_to current_user.email, account_path %></li>
+              <% end %>
```

Comment:
indentation is confusing.  unless/else/end should line up, as should the two <li>'s